pmd-core/src/main/java/net/sourceforge/pmd/lang/BaseLanguageModule.java

```diff
@@ -56,9 +57,9 @@ protected void addVersions(LanguageVersionHandler languageVersionHandler, boolea
             defaultVersion = languageVersion;
         }
     }
-
+    
     protected void addVersion(String version, LanguageVersionHandler languageVersionHandler, boolean isDefault) {
```

Comment:
shall we deprecate it and promote the experimental version?

Reply:
I think, we shouldn't remove the redundant overload on master, but deprecating it and removing it later is appropriate. Removing it abruptly would at least break the designer, though I don't know about other dependent projects

Reply:
true, we are compiling the designer against older pmd-core for release, and get a bumped pmd-core in runtime. I completely forgot about that one.